internal/venice-common/src/main/java/com/linkedin/venice/serialization/avro/AvroProtocolDefinition.java

```diff
@@ -147,7 +147,7 @@ public enum AvroProtocolDefinition {
   /**
    * Value schema for metadata system store.
    */
-  METADATA_SYSTEM_SCHEMA_STORE(29, StoreMetaValue.class),
```

Comment:
I think we should have [protocol] in the commit tag to notice that we upgrade protocol

Reply:
actually [compat]